cmd/dep/ensure.go

```diff
@@ -670,7 +670,7 @@ func (cmd *ensureCommand) runAdd(ctx *dep.Ctx, args []string, p *dep.Project, sm
 	}
 	sort.Strings(reqlist)
 
-	sw, err := dep.NewSafeWriter(nil, p.Lock, dep.LockFromSolution(solution), dep.VendorOnChanged)
+	sw, err := dep.NewSafeWriter(nil, nil, p.Lock, dep.LockFromSolution(solution), dep.VendorOnChanged)
```

Comment:
Shouldn't you pass `p.Manifest` and the appender here?

Reply:
@ibrasho 
oops. I am sorry, I missed to push the ensure.go changes.